port/win32_include/omrportptb.h

```diff
@@ -39,6 +39,7 @@
 #include "omrport.h"
 
 #define J9ERROR_DEFAULT_BUFFER_SIZE 256 /**< default customized error message size if we need to create one */
+#define J9ERROR_MAXIMUM_BUFFER_SIZE 0xFFFFFFFF /**< maximum customized error message size if we need to create one */
```

Comment:
Is this maximum buffer size correct for both 32 and 64 bit platforms? Doesn't the stdint library provide a max for unintptr_t?